resource/supported.go

```diff
@@ -307,7 +313,44 @@ func (a *AWS) elbs() (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	return output.LoadBalancerDescriptions, nil
+	loadBalancerNames := make([]string, len(output.LoadBalancerDescriptions))
+	loadBalancerNameMap := make(map[string]*elb.LoadBalancerDescription)
+	for i, desc := range output.LoadBalancerDescriptions {
+		loadBalancerNames[i] = *desc.LoadBalancerName
+		loadBalancerNameMap[*(desc.LoadBalancerName)] = desc
+	}
+
+	tagDescriptions, err := a.findElbTags(loadBalancerNames)
+	if err != nil {
+		return nil, err
+	}
+	loadBalancers := make([]*LoadBalancerDescription, len(output.LoadBalancerDescriptions))
+	for i, tagDesc := range tagDescriptions {
+		loadBalancer := loadBalancerNameMap[*(tagDesc.LoadBalancerName)]
+		loadBalancers[i] = &LoadBalancerDescription{*loadBalancer, tagDesc.Tags}
+	}
+	return loadBalancers, nil
+}
+
+func (a *AWS) findElbTags(elbNames []string) ([]*elb.TagDescription, error) {
```

Comment:
Thanks for your PR :+1: Not sure if you have seen that I have some generic function that tries to find tag fields by name (https://github.com/cloudetc/awsweeper/pull/47/files#diff-badd01dec4bab337623be10dca5e352cR136) in the output (https://github.com/cloudetc/awsweeper/blob/master/resource/resource.go#L81). 

Not sure if that will work for ELB, but I will have a look and come back to you.

Reply:
The elb tags are not returned in the first call and have to be requested separately.

Reply:
Ah, I see, makes sense.